jetty-security/src/main/java/org/eclipse/jetty/security/ConstraintSecurityHandler.java

```diff
@@ -741,14 +741,9 @@ public boolean checkPathsWithUncoveredHttpMethods()
         Set<String> paths = getPathsWithUncoveredHttpMethods();
         if (paths != null && !paths.isEmpty())
         {
-            ContextHandler.Context currentContext = ContextHandler.getCurrentContext();
-
-            for (String p : paths)
-            {
-                LOG.warn("{} has uncovered http methods for path: {}", currentContext, p);
-            }
-            if (LOG.isDebugEnabled())
-                LOG.debug("{} has uncovered http methods", currentContext, new Throwable());
+            LOG.warn("{} has paths with uncovered http methods: [{}]",
+                ContextHandler.getCurrentContext(),
+                String.join(", ", paths));
```

Comment:
There is no need for `String.join()` as `Set.toString()` already formats like you want (external brackets and commas to separate items).

Reply:
if we backport this to 9.4.x then removing the return value is a bad idea.
But for 10+ i'm fine with removing the return value.